src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -57,4 +57,24 @@ public function equals(BaseOrderItemInterface $item)
             && $item->getVariant() === $this->variant
         );
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments on order item and total tax of units.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
```

Comment:
Current adjustments system implementation isn't perfect, that's why we need this "duplication". We were thinking about introducing `AdjustmentsBag` that will have all these helper methods, but that is a random idea for eventual later refactoring.

Regarding this method I responded above :)